xwiki-platform-core/xwiki-platform-like/xwiki-platform-like-ui/src/main/resources/XWiki/Like/UserProfileUIX.xml

```diff
@@ -254,30 +165,49 @@
     </property>
     <property>
       <content>{{velocity}}
-=$escapetool.xml($services.localization.render("like.userprofile.menu"))=
-#set($columns = ["doc.title", "doc.location", "like"])
+=$escapetool.xml($services.localization.render('like.userprofile.menu'))=
 #set ($docTitleDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.title')))
 #set ($docLocationDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.location')))
 #set ($likesDisplayName = $escapetool.xml($services.localization.render('like.livetable.column.likes')))
-#set($columnsProperties = {
-  "doc.title" : { "displayName": $docTitleDisplayName, "type": "text", "size": 30, "link": "view", "sortable": false, "filterable": false },
-  "doc.location" : { "displayName": $docLocationDisplayName, "type": "text", "size": 30, "html": true, "sortable": false, "filterable": false },
-  "like" : { "displayName": $likesDisplayName, "type" : "number", "sortable": false, "filterable": false }
-})
-
-#set ($currentUser = $services.model.serialize($doc.documentReference, "default"))
-#set ($pageToCallReference = $services.model.resolveDocument("XWiki.Like.UserProfileUIX"))
-#set ($pageToCall = $xwiki.getDocument($pageToCallReference))
-#set ($queryParams = {
-  "user": $currentUser,
-  "livetable": "true",
-  "xpage": "plain",
-  "outputSyntax": "plain"
-})
-#set($options = {
-  'url': $pageToCall.getURL('get', $escapetool.url($queryParams))
+#set ($currentUser = $services.model.serialize($doc.documentReference, 'default'))
+#set ($liveDataConfig = {
+  'meta': {
+    'propertyDescriptors': [
+       {
+         'id': 'doc.title',
+         'sortable': false,
+         'filterable': false,
+         'name': $docTitleDisplayName
+       },
+       {
+         'id': 'doc.location',
+         'sortable': false,
+         'filterable': false,
+         'name': $docLocationDisplayName
+       },
+       {
+         'id': 'like',
+         'displayer': 'number',
+         'sortable': false,
+         'filterable': false,
```

Comment:
These 2 can be omitted, as properties need to be made sortable and filterable explicitly. For the doc title and location is different because the default configuration makes them sortable and filterable so we need to overwrite that I think.

Reply:
I can see ``sortable`` and ``filterable`` are still explicitly set to ``false`` for the ``like`` column. Why?